src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -117,20 +117,20 @@ private int WriteRunToErrorList(Run run, string logFilePath, Solution solution)
                 }
             }
 
-            if (run.Invocation != null)
+            if (run.Invocations != null && run.Invocations.Count > 0)
```

Comment:
If this code is populating the error list with the notifications, then you should loop over all the invocations, not just the first. It's true that there's no code yet to populate multiple invocations, but we should be able to _consume_ them.

Reply:
Right, obvious. Fixed.